src/pages/PassportRedemption/SignIn/uploadScreen.tsx

```diff
@@ -0,0 +1,302 @@
+    } catch (err) {
+      console.log(err);
+    }
+  };
+
+  return (
+    <>
+      <Column>
+        <SubTitle>{t('passport.labels.showReceipt')}</SubTitle>
+        <Column />
+        <Label htmlFor="crawl-merchant-name">
+          {t('passport.labels.crawlMerchant')}
+        </Label>
+        <CustomSymbolContainer className="allParticipatingSellers">
+          <InputField
+            name="crawl-merchant-name"
+            type="text"
+            onChange={(e) => {
+              e.preventDefault();
+              setParticipatingSeller(e.target.value);
+              setSelectedSeller(false);
+            }}
+            value={participatingSeller}
+            placeholder="Search Chinatown Businesses"
+          />
+        </CustomSymbolContainer>
+
+        {!!participatingSeller && !selectedSeller ? (
+          !!sellersSearchBar.length ? (
+            <SearchContainer>
+              {sellersSearchBar.map((merchant) => (
+                <SearchResult
+                  key={merchant.name + 'lny-crawl'}
+                  onClick={(e) => {
+                    e.preventDefault();
+                    setParticipatingSeller(merchant.name);
+                    setSelectedSeller(true);
+                    setParticipatingSellerId(merchant.id);
+                  }}
+                >
+                  {merchant.name}
+                </SearchResult>
+              ))}
+            </SearchContainer>
+          ) : (
+            <SearchContainer className="addPadding">
+              {t('passport.errors.validParticipatingSeller')}
+            </SearchContainer>
+          )
+        ) : null}
+
+        <Label htmlFor="bill-total">{t('passport.labels.billTotal')}</Label>
+        <CustomSymbolContainer className="billTotal">
+          <InputField
+            name="bill-total"
+            className="indent"
+            type="number"
+            onChange={(e) => {
+              e.preventDefault();
+              setBillTotal(e.target.value);
+            }}
+            value={billTotal}
+            min="10"
+            step=".01"
+          />
+          <SubTitle color="grey" spacing="none">
+            $10 minimum
+          </SubTitle>
+        </CustomSymbolContainer>
+
+        <ButtonTest>
+          {!receipt ? (
+            <>
+              <img src={uploadImageIcon} alt="upload-receipt-arrow-img" />
+              <p />
+              <span>{t('passport.labels.uploadReceipt')}</span>
+            </>
+          ) : (
+            <>
+              <SelectedImage
+                src={receiptFilePath}
+                alt="upload-receipt-arrow-img"
+              />
+              <ReplaceButton>
+                {t('passport.labels.reuploadReceipt')}
+              </ReplaceButton>
+            </>
+          )}
+          <input
+            type="file"
+            id="imageFile"
+            capture="environment"
+            accept="image/*"
+            onChange={(e) => uploadImage(e)}
```

Comment:
you can do onChange={uploadImage} and it would do the samething

Reply:
trueeeee